system/datatypes/peer_reviewed_article.php

```diff
@@ -489,6 +498,12 @@ protected function editAction(HyphaRequest $request) {
 		$this->xml->find(self::FIELD_NAME_METHOD)->setHtml($form->dataFor(self::FIELD_NAME_METHOD));
 		$this->xml->find(self::FIELD_NAME_SOURCES)->setHtml($form->dataFor(self::FIELD_NAME_SOURCES));
 
+		$filename = $form->dataFor(self::FIELD_NAME_FEATURED_IMAGE);
+		if ($filename) {
+			$filename = HyphaImage::ROOT_PATH . $filename;
```

Comment:
Why add the `ROOT_PATH` here? That only hurts portability (moving the hypha installation)? Leaving it out also prevents having to remove it again when inserting data into the form values later? Also, when using this filename, you'd normally pass it to the `HyphaFile` constructor, which also needs it *without* the `ROOT_PATH` prepended.

Reply:
fixed